pkg/common/cliprinter/protojson/protojson.go

```diff
@@ -0,0 +1,83 @@
+package protojson
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/spiffe/spire/pkg/common/cliprinter/errorjson"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+// Print prints one or more protobuf messages formatted as JSON
+func Print(msgs []proto.Message, stdout, stderr io.Writer) {
```

Comment:
Wonder if these print methods should return `error`.... if there is a problem writing to the output stream (e.g. a broken pipe), I don't think we want to exit with status 0.

Reply:
Hmm, that's a good point. Yes I will think about that one.

Reply:
If a caller receives an error .. we expect the caller to immediately exit non-zero? Or we expect it to try to further print or report the error somehow?

Reply:
Good question, I'd probably lean on best-effort printing to stderr and exit non-zero.

Reply:
How would you feel about something like `MustPrint` which does this and tries to exit non-zero rather than panic?

I think ideally, the caller never has to (or should feel the need to) resort to use of `fmt.Printf` and friends, since that's when things tend to drift... and we should try to capture as much of it as we can in this cliprinter. We could return ok, but then the caller may feel urged to say something about it? Either way, I feel it's a good idea to let cliprinter handle the error message emission

Reply:
That sounds fine to me.